### PR TITLE
Change stats color when type /r

### PR DESCRIPTION
**Pull request by Toby1710**
It separates the stats message from other messages.
image

### CODE REVIEW

File: mods/ctf/ctf_stats/chat.lua [hunk 1]
```diff
@@ -31,22 +31,30 @@ local function return_as_chat_result(to, target)
...
+			            minetest.colorize("#63d437", " | K/D: ") ..
+                        minetest.colorize("#ffea00", math.floor(kd * 10) / 10 ..
+			            minetest.colorize("#63d437", "\nBounty kills: ") ..
+                        minetest.colorize("#ffea00", stat.bounty_kills ..
+			            minetest.colorize("#63d437", " | Captures: ") ..
+                        minetest.colorize("#ffea00", stat.captures ..
+			            minetest.colorize("#63d437", " | Attempts: ") ..
+                        minetest.colorize("#ffea00", stat.attempts ..
+			            minetest.colorize("#63d437", "\nScore: ") ..                         
+                        minetest.colorize("#ffea00", math.floor(stat.score))))))))
```
Comment (JostP) [inline mods/ctf/ctf_stats/chat.lua:57]: Why so many  closing brackets?

Comment (AnyBlue) [reply mods/ctf/ctf_stats/chat.lua:57]: > Why so many closing brackets?

Oh, yeah... It wasn't working for me

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please fix [LuaCheck](https://github.com/MT-CTF/capturetheflag/pull/759/checks?check_run_id=1744107107) warnings.

**Comment by AnyBlue** [comment]
> Please fix [LuaCheck](https://github.com/MT-CTF/capturetheflag/pull/759/checks?check_run_id=1744107107) warnings.

I know

File: mods/ctf/ctf_stats/chat.lua [hunk 2]
```diff
@@ -31,22 +31,30 @@ local function return_as_chat_result(to, target)
...
 		end
 		result = result ..
-			"Kills: " .. stat.kills ..
-			" | Deaths: " .. stat.deaths ..
-			" | K/D: " .. math.floor(kd * 10) / 10 ..
-			"\nBounty kills: " .. stat.bounty_kills ..
-			" | Captures: " .. stat.captures ..
-			" | Attempts: " .. stat.attempts ..
-			"\nScore: " .. math.floor(stat.score)
+			minetest.colorize("#63d437", "Kills: ") ..
```
Comment (Panquesito7) [inline mods/ctf/ctf_stats/chat.lua:44]:
```suggestion
			minetest.colorize("#63d437", "Kills: ") ..
```

File: mods/ctf/ctf_stats/chat.lua [hunk 3]
```diff
@@ -31,22 +31,30 @@ local function return_as_chat_result(to, target)
...
+                        minetest.colorize("#ffea00", stat.deaths ..
+			minetest.colorize("#63d437", " | K/D: ")) ..
+                        minetest.colorize("#ffea00", math.floor(kd * 10) / 10 ..
+			minetest.colorize("#63d437", "\nBounty kills: ")) ..
+                        minetest.colorize("#ffea00", stat.bounty_kills ..
+			minetest.colorize("#63d437", " | Captures: ") ..
+                        minetest.colorize("#ffea00", stat.captures ..
+			minetest.colorize("#63d437", " | Attempts: ")) ..
+                        minetest.colorize("#ffea00", stat.attempts ..
+			minetest.colorize("#63d437", "\nScore: ")) ..
```
Comment (Panquesito7) [inline mods/ctf/ctf_stats/chat.lua:56]:
```suggestion
			minetest.colorize("#63d437", "\nScore: ")) ..
```

**Review by moreavy — APPROVED** [review]
Didn't test but the Idea and the screenshot looks good. All Fine 👍🏾

**Pull request merged**